test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -364,6 +364,14 @@ public abstract class OpenSearchIntegTestCase extends OpenSearchTestCase {
     private static OpenSearchIntegTestCase INSTANCE = null; // see @SuiteScope
     private static Long SUITE_SEED = null;
 
+    public static List<String> codecs = List.of(
```

Comment:
[minor]: CODECS in caps for static field.

Reply:
ah right, dont know I missed this! thankss. I've updated!